src/metabase/models/database.clj

```diff
@@ -34,23 +34,67 @@
 
 (models/defmodel Database :metabase_database)
 
+
+(defn- schedule-tasks!
+  "(Re)schedule sync operation tasks for DATABASE. (Existing scheduled tasks will be deleted first.)"
+  [database]
+  (try
+    (require 'metabase.task.sync-databases)
+    ((resolve 'metabase.task.sync-databases/schedule-tasks-for-db!) database)
+    (catch Throwable e
+      (log/error "Error scheduling tasks for DB:" (.getMessage e) "\n")
```

Comment:
FWIW @senior I did add a separate line to log the stacktrace below, but at a lower log level. Logging the entire stacktrace was getting annoying so I made the error message `error` and the stacktrace `debug`